src/tools/archiveTools.cpp

```diff
@@ -99,6 +88,71 @@ std::string getMetaPublisher(const zim::Archive& archive) {
   return getMetadata(archive, "Publisher");
 }
 
+std::string getMetaFlavour(const zim::Archive& archive) {
+  return getMetadata(archive, "Flavour");
+}
+
+std::string getArchiveId(const zim::Archive& archive) {
+  std::ostringstream s;
+  s << archive.getUuid();
+  return s.str();
+}
+
+std::string getArchiveOrigId(const zim::Archive& archive) {
+  std::string value = getMetadata(archive, "startfileuid");
+  if (value.empty()) {
+    return "";
+  }
+  std::string id = value;
+  std::string origID;
+  std::string temp = "";
+  unsigned int k = 0;
+  char tempArray[16] = "";
+  for (unsigned int i = 0; i < id.size(); i++) {
+    if (id[i] == '\n') {
+      tempArray[k] = atoi(temp.c_str());
+      temp = "";
+      k++;
+    } else {
+      temp += id[i];
+    }
+  }
+  origID = (std::string) zim::Uuid::generate(tempArray);
+  return origID;
+}
```

Comment:
I agree with you. If it's not of use to us right now, we should remove it. And to be honest, it seemed more like belonging to libzim rather than libkiwix.